src/plugins/deps/geomap-lib.js

```diff
@@ -192,8 +192,10 @@ var componentName = "wb-geomap",
 		// Add map layers
 		this.addMapLayers();
 
-		// If an extent was configured, fit the map to it
-		if ( viewOptions.extent ) {
+		// If an extent was configured, fit the map to it (use mapExtent property, then basemap.extent)
+		if ( this.settings.mapExtent ) {
```

Comment:
Could we do simply `extent` instead of `mapExtent` since we are already in the context of a map? Or is it because the word is already reserved for something else?

Reply:
Since "extent" seemed to already be used and, since I don't know the plugin throughout, I didn't want to create confusion or conflict between with this new property. However, I am very open to any other name. TBH, I am not a fan of this property name either, but I couldn't come up with a better name myself.

Reply:
@Garneauma Ok then, even though I am not a fan of the current name either, the rationale makes sense so let's keep `mapExtent`.